cms/templates/studio_xblock_wrapper.html

```diff
@@ -14,6 +14,7 @@
 collapsible_class = "is-collapsible" if xblock.has_children else ""
 label = xblock.display_name_with_default or xblock.scope_ids.block_type
 messages = xblock.validate().to_json()
+is_unit = xblock._is_unit()
```

Comment:
A "private" method (starting with an underscore) shouldn't be called outside of the file.